readme.md

```diff
@@ -26,7 +28,7 @@ interval.toPostgres()
 
 ##### pgInterval
 
-*Required*  
+*Required*
```

Comment:
Please make sure this trailing whitespace stays, it's important to proper formatting